.gitignore

```diff
@@ -4,6 +4,7 @@ dist
 *.egg-info
 *.py[cod]
 tmp_*
+node_modules
```

Comment:
It's already in `catalog/.gitignore`, though? You can move it here - but then delete it from the other one.

Reply:
Yeah done